### ISSUE TITLE
Node module version from package-lock.json

### ISSUE THREAD

**Issue by vonas** [badges/shields#4693]
:clipboard: **Description**

[![npm](https://img.shields.io/npm/v/mermaid?color=teal&label=mermaid&logo=npm)](https://img.shields.io/npm/v/mermaid?color=teal&label=mermaid&logo=npm)
Shows the exact version of a package found in `package-lock.json`.

:link: **Data**

Retrieving the data should function pretty much the same way as it already works for `package.json`.

:microphone: **Motivation**

This badge should co-exist with the already existing `package.json` badge. The only difference would be that it gets the module's version from `package-lock.json` which allows you to show on which exact version of a package your code currently depends on.

**Comment by paulmelnikow** [comment]
Agreed this is a good idea. We have something similar for Pipfile.lock. (Actually we only look in Pipfile.lock, not in Pipfile.) Would be happy to have a PR for this! If you're interested in working on this and need help, feel free to post in this thread and a maintainer can answer your questions.

**Comment by paulmelnikow** [comment]
Sorry, to clarify, this would be for dependencies, correct?

**Comment by vonas** [comment]
That's correct.

I'll have a look at implementing this myself!
